handlers/bounty_test.go

```diff
@@ -485,6 +507,15 @@ func TestGetBountyByCreated(t *testing.T) {
 		handler.ServeHTTP(rr, req)
 
 		assert.Equal(t, http.StatusOK, rr.Code)
+
+		var responseData []db.BountyResponse
+		err = json.Unmarshal(rr.Body.Bytes(), &responseData)
+		if err != nil {
+			t.Fatalf("Error decoding JSON response: %s", err)
+		}
+
+		assert.NotEmpty(t, responseData)
+		assert.Len(t, responseData, 1)
```

Comment:
noticed that we dont assert the exact value here

Reply:
@kevkevinpal [Added](https://github.com/stakwork/sphinx-tribes/pull/1544/commits/9305d7298863a25d8377daa60e4e3d50031b0496#diff-f47eb74033ab155c9b6163240180949abb75b2fd4a1dcd61d0a49d0278500aeaR519-R525) additional assertions to validate the response